utils/mosh/patches/disable-utf8-check.patch

```diff
@@ -0,0 +1,17 @@
+--- a/src/util/locale_utils.cc
++++ b/src/util/locale_utils.cc
+@@ -86,10 +86,10 @@ const char *locale_charset( void )
+ 
+ bool is_utf8_locale( void ) {
+   /* Verify locale calls for UTF-8 */
+-  if ( strcmp( locale_charset(), "UTF-8" ) != 0 &&
+-       strcmp( locale_charset(), "utf-8" ) != 0 ) {
+-    return false;
+-  }
```

Comment:
what's the purpose of this patch? it's unmentioned in your commit notes.

Reply:
This patch is also not needed. Removed it.